packages/components/inputs/localized-multiline-text-input/src/localized-multiline-text-input.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import type { Meta, StoryFn } from '@storybook/react';
+import LocalizedMultilineTextInput, {
+  TLocalizedMultilineTextInputProps,
+} from './localized-multiline-text-input';
+import { useState } from 'react';
+
+const meta: Meta<typeof LocalizedMultilineTextInput> = {
+  title: 'Form/Inputs/LocalizedMultilineTextInput',
+  // @ts-ignore
+  component: LocalizedMultilineTextInput,
+};
+export default meta;
+
+type Story = StoryFn<typeof LocalizedMultilineTextInput>;
+
+export const BasicExample: Story = ({
+  defaultExpandMultilineText,
+  defaultExpandLanguages,
+  ...args
+}: TLocalizedMultilineTextInputProps) => {
+  const [value, setValue] = useState({
+    en: 'Horse\nCow\nDuck',
+    de: 'Pferd\nKuh\nEnte',
+    'nan-Hant-TW': '马\n奶牛\n鸭子',
```

Comment:
nit, Hant-TW is traditional, these translations are in simplified
```suggestion
    'nan-Hant-TW': '馬\n奶牛\n鴨子',
```